blocks/index.js

```diff
@@ -22,6 +22,7 @@ export { default as BlockEdit } from './block-edit';
 export { default as BlockIcon } from './block-icon';
 export { default as ColorPalette } from './color-palette';
 export { default as Editable } from './rich-text/editable';
+export { default as ImagePlaceHolder } from './image-placeholder';
```

Comment:
Given the naming of the directory, I'd expect this to be camel-cased as: `ImagePlaceholder`

Reply:
Agreed, @aduth. The most recent commit changes `ImagePlaceHolder` to `ImagePlaceholder` (everywhere).